### ISSUE TITLE
Contentful

### ISSUE THREAD

**Issue by anthanh** [junaid33/opensource.builders#205]
CMS platform

**Comment by jcklpe** [comment]
@anthanh  any special features on this that sets it apart from something like Strapi, Wordpress, or Directus?

**Comment by davidak** [comment]
>More than a headless CMS, Contentful is the API-first content management platform to create, manage and publish content on any digital channel

So like https://headlesscms.org/

**Comment by jcklpe** [comment]
@davidak Yeah from looking over it, it seems like it's another headless cms. I only have personal experience with Wordpress Headless CMS, but wasn't sure if this one should be collapsed in with the #217  ticket that's already been logged or not. It can be hard sometimes to compare headless CMS features one to one since they can vary so much.

**Comment by davidak** [comment]
@jcklpe Gatsby is a Static Site Generator you can use standalone to generate a site from markdown files, or in combination with a headless CMS. so you have the intuitive webinterface of wordpress and the performance/security of static sites.

https://www.staticgen.com/

when you sort the headlesscms overview after twitter followers, Contentful is actualy the most popular, so it makes sense to show it and alternatives for it. you can also link the headlesscms page...

![Screenshot from 2020-04-08 05-15-12](https://user-images.githubusercontent.com/91113/78740774-f6536e80-7957-11ea-84ae-9c288c96300a.png)

**Comment by jcklpe** [comment]
@davidak  oh yeah sorry I was thinking of netlify. That's also opensource though. 

Looking over the headlesscms website, the only closed source ones I've really heard of are contentful, and butter cms. I thought craft was headless but I guess not. And yeah contentful seems by and away the most popular. 

As far as open source alternatives, I haven't tried, but I've been interested in checking out Directus and cockpit.

**Comment by agustif** [comment]
PrimeCMS

**Comment by anthanh** [comment]
https://strapi.io/

https://directus.io/
